README.md

```diff
@@ -11,6 +11,8 @@ We're going to have Users, Notes, and a viewers join table, which gives users re
 
 ## Instructions
 
+This lab uses Postgres. Make sure you have the Postgres app installed (`brew install postgres`), and running before you run the tests. From there, you'll need to create the database with `rake db:create`. 
```

Comment:
We should add a note here to follow the post-install instructions from brew (or to bring them up again with `brew info postgres`) so that we're super explicit.